server/src/main/java/com/microsoft/java/bs/core/internal/managers/BuildTargetManager.java

```diff
@@ -110,13 +112,21 @@ private List<String> getBuildTargetTags(boolean hasTests) {
   }
 
   private void setJvmBuildTarget(GradleSourceSet sourceSet, BuildTarget bt) {
+    DefaultJavaExtension javaExtension = ConversionUtils.toJavaExtension(
+        sourceSet.getExtensions().get("java"));
```

Comment:
Use `JavaLanguageModelBuilder#LANGUAGE_ID` instead of "java"?

Reply:
The project `plugin` is not on the classpath of the `server` project.

Put this const somewhere in the `model` project?

Reply:
Oops - I thought it was in Model, not Plugin.  Up to you - it's not a big issue